spec/actionizer/inputs_spec.rb

```diff
@@ -276,6 +276,21 @@ def call; end
           end
         end
 
+        context 'and the type specified is a parent class' do
+          context 'and the arg passed is a descendent' do
+            it 'succeeds' do
+              expect(dummy_class.call(foo: Integer)).to be_success
+            end
+          end
+          context 'and the arg passed is not a descendent' do
```

Comment:
You're gonna hate me, but can you add a blank line between these contexts? They're line two paragraphs being squished together like this.

Reply:
Okay.